conda_build.recipe/build.sh

```diff
@@ -3,3 +3,4 @@
 $PYTHON setup.py install
 
 cp bdist_conda.py ${PREFIX}/lib/python${PY_VER}/distutils/command
+install bin/* ${PREFIX}/bin
```

Comment:
And yes, it does look like setup.py does do this (and spams to the screen that it is installing).  I wonder if the issue is a missing `hash -r` call as the testing env does not get properly activated.  Does `install` do a bit more magic under the hood?